tutor/src/components/task-plan/index.cjsx

```diff
@@ -56,9 +58,13 @@ EventShell = React.createClass
 PlanShell = React.createClass
   displayName: 'PlanShell'
 
```

Comment:
will merge for now to take more time for a more global fix, which will require much more testing